tasks/configuration/Linux.yml

```diff
@@ -28,6 +28,7 @@
     mode: 0755
   become: True
   when:
+    - solr_auth_configure
     - solr_auth_type is defined
```

Comment:
Should we remove it?
`solr_auth_type is defined` in defaults so it be always True